plenum/test/consensus/order_service/test_can_send_3pc.py

```diff
@@ -35,15 +28,15 @@ def test_can_send_3pc_batch_old_pp_seq_no_for_view(primary_orderer, mode):
     assert not primary_orderer.can_send_3pc_batch()
 
 
-@pytest.mark.parametrize('initial_seq_no', [0, 3, 8, 13])
+@pytest.mark.parametrize('initial_seq_no', [1, 3, 8, 13])
```

Comment:
Why we no longer test against seq_no 0?

Reply:
Because of new wait-till-first-batch-ordered-in-view logic.
Although this stashing and checks are done for view>0 only, the tests are parametrized to be run with views>0, and if initial_seq_no=0. then we may fall to this situation. There are another new tests added to deal with this.